dfastmi/io/CelerObject.py

```diff
@@ -0,0 +1,15 @@
+from abc import ABC
+from typing import List
+
+
+class CelerObject(ABC):
```

Comment:
don't forget documentation.

Reply:
will add

Reply:
will be done in celer object refactoring issue